### PR TITLE
Feature: Add Community Builders Page

### LINKED ISSUE #1: [FEATURE] Start the Community Builders landing page
Thanks @Simperfy for helping on this design. Only Team Web3 can work with this issue as for their recognition on contributing to @web3community & @dev-protocol community. Feel free to get this green square📗. 

We will launch the Community Builders on Monday, so hopefully it can be done on Sunday night (GMT+8). 

If no one work here on Saturday, I'll be taking this issue.

## Implementation

The design is here

https://www.figma.com/file/ZWhbYOGvx88xqLmRhCcwQX/Community-Builders-Website?node-id=0%3A1

#### Tech Stacks
- TailwindUI
- Any tech stack will do (Next.js, React, Astro)
- You can suggest below on what will u use

### PR DESCRIPTION
**Pull request by AbhinavMV**
Please let me know if any changes are needed.
You can navigate using navbar to the page. Community -> Builders Program
Dark Mode:
image

Light Mode:
image

There are many yaml files from devprotocol code. 
Do you want me to remove them? (If so please also tell me which are those files as i have not played around with them yet. 😋)

### CODE REVIEW

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+          pr-message: '<h1>Great to see you here!</h1> <br><br> Welcome to the community :tada: <br> Thank you for your valuable contributions! 🚀 We appreciate each and every contribution! 😄 <br><br> <h2>Join our amazing community 📢</h2> <br><br> Join our community to learn more about Blockchain, Web3, Dapps, and so much more 💻 <br><br> Here you can find all of our socials! 🐦 Share them with your friends and everyone you know. <br><br> - <a href="https://discord.gg/VwJp4KM">Discord server</a> <br> - <a href="http://twitter.com/devprtcl">Twitter</a><br> - <a href="https://www.linkedin.com/company/dev-protocol">LinkedIn</a><br> -  <br> - <a href="https://t.me/devprotocol">Telegram</a><br> - a href="https://www.youtube.com/channel/UCN7m74tFgJJnoGL4zk6aJ6g">YouTube</a><br> <a href="https://www.facebook.com/DevProtocol.xyz/">Facebook</a>- <br><br> We hope to see you there! 😁'
```
Comment (vinzvinci) [inline .github/workflows/first-interaction.yml]: I will update this.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,52 @@
-# Website for the Community Builders
```
Comment (vinzvinci) [inline README.md:1]: Let me do the README.

File: src/components/CommunityBuilders/Faq.astro [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<section
+  class="
+    p-5 my-6 md:-mt-10
+  "
+>
+  <div class="w-auto items-center flex flex-col space-y-6 text-center md:text-left">
+  <div class="grid grid-cols-2 gap-6">
+    <div class="col-span-2 md:col-span-1 mx-8 sm:mx-16 space-y-2">
+        <h4 class="font-semibold font-display text-3xl leading-relaxed">Frequently asked questions</h4>
+        <p>Can’t find the answer you’re looking for? Reach out to our support team.</p>
```
Comment (vinzvinci) [inline src/components/CommunityBuilders/Faq.astro]: Include `<a href="https://discord.gg/VwJp4KM">support</a>`

Comment (AbhinavMV) [reply src/components/CommunityBuilders/Faq.astro]: Done 👍

**Comment by vinzvinci** [comment]
Check my dm on discord.

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Pull request merged**